static/javascript/index.js

```diff
@@ -171,6 +280,14 @@ function removeNoteFromDB(event){
 
 }
 
+/// Sign-Out with Facebook //////
+function sign_out(){
+    console.log("signed_out running")
+     window.location = "/";
+}
```

Comment:
this should be called `afterSignOut()` and it should use `window.location.replace("/")` or maybe [`window.location.assign("/")`](https://developer.mozilla.org/en-US/docs/Web/API/Location/assign)